IKU/IKU/Views/SelectPhotoViewController.swift

```diff
@@ -121,6 +112,26 @@ final class SelectPhotoViewController: UIViewController {
         }
     }
     
+    private func popAndDisapperAnimation(of view: UIView) {
+        view.alpha = 1
+        view.transform = CGAffineTransform.identity.scaledBy(x: 0.5, y: 0.5)
+        UIViewPropertyAnimator.runningPropertyAnimator(
+            withDuration: 0.2,
+            delay: 0,
+            options: .curveEaseInOut) {
+                view.transform = CGAffineTransform.identity
+            } completion: { _ in
+                UIView.transition(with: view, duration: 1) {
+                    view.alpha = 0
+                }
+            }
+    }
+    
+    func prepareURL(url: URL) {
```

Comment:
p2)

현재 이 함수는 아무도 부르고 있지 않아 보입니다.
직전 View로 부터 url을 받지 않고 이 View로 넘어올 수 없도록, init()을 만들어서 작업하면 어떨까요?
convenience init을 제안합니다.
```suggestion
    convenience init(withVideoURL url: URL)
```

Reply:
@sustainable-git
이전 VC에서 SelectPhotoViewController를 생성 후 [호출](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-IKU/pull/50/files#diff-54004a3a41893a06134decf7651752be61f9a8c9eba26dc5de96a2c66a50088fR228)하고 있기는 합니다!